# Z Units.
# None.
#uint8 Z_NONE = 0
# Depth.
#uint8 Z_DEPTH = 1
# Altitude.
#uint8 Z_ALTITUDE = 2
# Height.
#uint8 Z_HEIGHT = 3

std_msgs/Float64 value
std_msgs/UInt8 z_units
